spec/classes/redis_spec.rb

```diff
@@ -679,7 +679,7 @@ class { 'redis::globals':
 
         it do
           if facts[:operatingsystem] == 'Ubuntu' && facts[:operatingsystemmajrelease] == '16.04'
-            is_expected.not_to contain_file(config_file_orig).with_content(%r{protected-mode})
+            is_expected.not_to contain_file(config_file_orig).with_content(%r{^protected-mode$})
```

Comment:
The regex isn't really needed. In fact, I'd say it's better not to be strict. It's expecting that `protected-mode` isn't present at all, regardless of where in the file it shows up.

Reply:
I would say it's better to be strict.
If we only set variables, without touching documentation inside the file, so people can use them for reference then 
`# protected-mode example` would be shown somewhere which is caught by the previous regexp and the test fails